pkg/commands/apply/apply.go

```diff
@@ -319,6 +327,32 @@ func (c *ApplyCommand) Process(ctx context.Context) (merr error) {
 	return merr
 }
 
+func (c *ApplyCommand) resolveGitHubLogURL(ctx context.Context) string {
```

Comment:
I wonder if we should create a helper function on the github client to generate this URL

Reply:
Ideally generate the URL and we can format the link in the caller

Reply:
SGTM, some de-duping of the copy-pasted code seems beneficial.

Reply:
Done